src/tests/unit/tests/DetailsView/components/export-dialog.test.tsx

```diff
@@ -4,8 +4,13 @@ import { IMock, Mock, MockBehavior, Times } from 'typemoq';
 
 import { DetailsViewActionMessageCreator } from '../../../../../DetailsView/actions/details-view-action-message-creator';
 import { ExportDialog, ExportDialogProps } from '../../../../../DetailsView/components/export-dialog';
+import { ReportFileNameGenerator } from '../../../../../DetailsView/reports/components/report-file-name-generator';
 
 describe('ExportDialog', () => {
+    const reportFileName = 'THE REPORT FILE NAME';
+    const reportFileNameGeneratorMock = Mock.ofType(ReportFileNameGenerator);
```

Comment:
you probably want to use `Mock.ofType<ReportFileNameGenerator>();` instead... works fine with interfaces and types, don't need a concrete class

Reply:
I removed that mock so the comment isn't relevant anymore. I noted your feedback though, thanks.

---
In reply to: [267545106](https://github.com/Microsoft/accessibility-insights-web/pull/398#discussion_r267545106) [](ancestors = 267545106)